src/dbally/audit/event_handlers/base.py

```diff
@@ -10,37 +10,44 @@
 
 class EventHandler(Generic[RequestCtx, EventCtx], ABC):
     """
-    Base event handler interface.
+    A base class that every custom handler should inherit from
     """
 
     @abc.abstractmethod
     async def request_start(self, user_request: RequestStart) -> RequestCtx:
         """
-        Log the start of the request.
+        Function that is called at the beginning of every `Collection.ask` execution.
 
         Args:
-            user_request: The start of the request.
+            user_request: Object containing name of collection and asked query
+
+        Returns:
+            Context that can be of `Any` type, which is passed to the future callbacks
```

Comment:
`Any` is part of static type analysis in Python, which our users doesn't necessarily have to use and understand.

I would explain this in words. And probably I wouldn't even say that the method can return "any type" because the type has to be consistent for a given implementation (that's why it is defined here as `RequestCtx`, not `Any`). I would probably describe it as an "implementation-specific request context object".